internal/backup.go

```diff
@@ -1,5 +1,7 @@
 package internal
 
+// 'Sentinel' mean a 'file' with meta/external-info about backup (in some cases)
```

Comment:
Move this to `backup_sentinel_dto.go`?

Reply:
We can add something more detailed:

WAL-G stores information about single backup in the following files:
1. Sentinel file - contains useful metadata, such as backup start time, backup size, etc.
2. (only in Postgres) Metadata file  - Postgres sentinel files can be quite large (> 1GB), so the metadata file is useful for the quick fetch of backup-related information.

Reply:
added this description to backup.go, fixed